UIViewController-DisplayInDrawer/Classes/UIViewController+PresentDrawerController.swift

```diff
@@ -241,7 +250,7 @@ extension UIViewController {
 }
 
 private extension UIView {
-    func addContainerView(for drawerConfiguration: DrawerConfiguration, cornerRadius: CGFloat, bottomPaddingHeight: CGFloat) -> UIView {
+    func addContainerView(for drawerConfiguration: DrawerConfiguration, cornerRadius: CGFloat, bottomPaddingHeight: CGFloat, withBackground: UIColor?) -> UIView {
```

Comment:
withBackground seems weird, simple `backgroundColor` might make better sense here and also at the calling site